slimta/smtp/datareader.py

```diff
@@ -84,7 +83,7 @@ def handle_finished_line(self):
                 self.EOD = i
 
             # Remove an initial period on non-EOD lines as per RFC 821 4.5.2.
-            elif line[0] == '.':
+            elif line[0:1] == b'.':  # line[0] is an integer
```

Comment:
Ok, will remove it, I though `line[0:1] != line[0]` is not obvious at all for someone reading the code, it suprised me at first actually.